.github/workflows/release.yaml

```diff
@@ -1,24 +1,25 @@
+# This workflow is triggered when a tag v* is pushed to the repository
+
+# To release to npm, make sure you incremented the version in package.json
+# and then run `git tag vx.x.x` and `git push origin vx.x.x`
 name: npm-publish
 on:
     push:
-        branches:
-            - main
+        tags:
+            - "v*"
+    workflow_dispatch:
+
 jobs:
-    npm-publish:
-        name: npm-publish
+    publish:
         runs-on: ubuntu-latest
         steps:
-            - name: Checkout repository
-              uses: actions/checkout@v2
-            - name: Publish if version has been updated
```

Comment:
Don't we want to keep this?

Reply:
I mean, a check that npm version was incremented

Reply:
Nope, I am taking a more lightweight approach to publishing to npm - as seen in GitHub documentation [1].

Afaik, this action I deleted does some magic to recognise a version change in the `package.json` and triggers an npm publish - which is nice, but didn't previously worked.

My proposal is to trigger the publishing when pushing a tag or manually by clicking the run button in github, so no more magic.

[1] https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry